INFO:root:Attempting to retrieve weather for London.
INFO:root:Retrieved city weather data: ['London', 6.19, 'Clear sky', 'static/icons/01d.png'].
INFO:root:Attempting to retrieve weather for Manchester.
INFO:root:Retrieved city weather data: ['Manchester', 7.06, 'Overcast clouds', 'static/icons/04d.png'].
INFO:root:Attempting to retrieve weather for Paris.
INFO:root:Retrieved city weather data: ['Paris', 5.14, 'Clear sky', 'static/icons/01d.png'].
INFO:root:Attempting to retrieve weather for Lecce.
INFO:root:Retrieved city weather data: ['Lecce', 8.67, 'Scattered clouds', 'static/icons/03d.png'].
INFO:root:Attempting to retrieve weather for Budapest.
INFO:root:Retrieved city weather data: ['Budapest', 4.42, 'Scattered clouds', 'static/icons/03d.png'].
INFO:root:Attempting to retrieve weather for Berlin.
INFO:root:Retrieved city weather data: ['Berlin', 1.54, 'Clear sky', 'static/icons/01d.png'].
INFO:root:Attempting to retrieve weather for Prague.
INFO:root:Retrieved city weather data: ['Prague', 0.39, 'Few clouds', 'static/icons/02d.png'].
INFO:root:Attempting to retrieve weather for Bratislava.
INFO:root:Retrieved city weather data: ['Bratislava', 1.57, 'Few clouds', 'static/icons/02d.png'].
INFO:root:Attempting to retrieve weather for Palermo.
INFO:root:Retrieved city weather data: ['Palermo', 10.43, 'Overcast clouds', 'static/icons/04d.png'].
INFO:root:Attempting to retrieve weather for Tokyo.
INFO:root:Retrieved city weather data: ['Tokyo', 7.39, 'Broken clouds', 'static/icons/04n.png'].
INFO:root:Attempting to retrieve weather for Langkawi.
INFO:root:Retrieved city weather data: ['Langkawi', 28.07, 'Few clouds', 'static/icons/02n.png'].
INFO:root:Attempting to retrieve weather for Milton Keynes.
INFO:root:Retrieved city weather data: ['Milton Keynes', 5.54, 'Clear sky', 'static/icons/01d.png'].
INFO:root:Attempting to retrieve weather for sszliapjxrhvyvt.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for flnoiqljkcpcwec.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for txklkluyulfomyl.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for vdwdlekomtsbrsl.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for tabfoodpgoqkzii.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for qjhnmduwydadxgz.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for puiixdvchhobxpe.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for rjvurkstoaqbicn.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for qjgeetwijzaiasu.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for tglmkzxcdsdkfmq.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for eruqtleddozucqx.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for ulwblusnzjerjje.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for zsmytvvlrayfbkr.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for gzxuxvqyivebrvb.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:root:Attempting to retrieve weather for dikywiysluxyoun.
ERROR:root:This city name is invalid. Please try again.
Traceback (most recent call last):
  File "/Users/harryjulian/Projects/weather-api/weather_app/app.py", line 45, in home
    raise InvalidCityException
weather_app.app.InvalidCityException
INFO:werkzeug:[31m[1mWARNING: This is a development server. Do not use it in a production deployment. Use a production WSGI server instead.[0m
 * Running on http://127.0.0.1:5000
INFO:werkzeug:[33mPress CTRL+C to quit[0m
INFO:werkzeug:127.0.0.1 - - [08/Feb/2023 12:41:08] "GET / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Feb/2023 12:41:08] "[33mGET /favicon.ico HTTP/1.1[0m" 404 -
INFO:root:Attempting to retrieve weather for Manchester.
INFO:root:Retrieved city weather data: ['Manchester', 7, 'Overcast clouds', 'static/icons/04d.png'].
INFO:werkzeug:127.0.0.1 - - [08/Feb/2023 12:41:12] "POST / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Feb/2023 12:41:12] "[36mGET /static/icons/04d.png HTTP/1.1[0m" 304 -
INFO:root:Attempting to retrieve weather for Milton Keynes.
INFO:root:Retrieved city weather data: ['Milton Keynes', 5.55, 'Clear sky', 'static/icons/01d.png'].
INFO:werkzeug:127.0.0.1 - - [08/Feb/2023 12:41:17] "POST / HTTP/1.1" 200 -
INFO:werkzeug:127.0.0.1 - - [08/Feb/2023 12:41:17] "[36mGET /static/icons/01d.png HTTP/1.1[0m" 304 -
